database/it1_report.json

```diff
@@ -0,0 +1,103 @@
+{
```

Comment:
This was not updated with the latest return, but the changes are minor, so I don't think it should affect FE